### PR TITLE
[docs]Add topic-level policy config

### PR DESCRIPTION
**Pull request by Jennifer88huang**
### Motivation
Topic-level policies are available since Pulsar 2.7.0.

### Modifications

Add how to configure topic-level policies.

Related to #9161

### CODE REVIEW

File: site2/docs/reference-pulsar-admin.md [hunk 1]
```diff
@@ -1759,13 +1759,20 @@ Options
...
 ## `topics`
-Operations for managing Pulsar topics (both persistent and non persistent)
+Operations for managing Pulsar topics (both persistent and non-persistent). 
 
 Usage
 ```bash
 $ pulsar-admin topics subcommand
 ```
 
+From Pulsar 2.7.0, some namespace level policies are available on topic level. To enable topic level policy in Pulsar, you need to configure the following parameters in the `broker.conf` file.
```
Comment (Huanli-Meng) [inline site2/docs/reference-pulsar-admin.md:1769]:
```suggestion
From Pulsar 2.7.0, some namespace-level policies are available on topic level. To enable topic-level policies in Pulsar, you need to configure the following parameters in the `broker.conf` file. 
```
same comments for docs of other releases.

**Review by Huanli-Meng — COMMENTED** [review]
Leave my comments. PTAL

File: site2/docs/reference-pulsar-admin.md (hunk 1 above)
Comment (Jennifer88huang) [reply site2/docs/reference-pulsar-admin.md]: Updated, thank you.

**Comment by freeznet** [comment]
LGTM

File: site2/docs/reference-pulsar-admin.md [hunk 2]
```diff
@@ -1815,6 +1822,36 @@ Subcommands
...
+* `get-compaction-threshold`
+* `set-compaction-threshold`
+* `remove-compaction-threshold`
+* `get-offload-policies`
+* `set-offload-policies`
+* `remove-offload-policies`
+* `get-max-unacked-messages-per-subscription`
+* `set-max-unacked-messages-per-subscription`
+* `remove-max-unacked-messages-per-subscription`
+* `get-max-unacked-messages-per-consumer`
```
Comment (freeznet) [inline site2/docs/reference-pulsar-admin.md:1846]: `get-max-unacked-messages-on-consumer`

File: site2/docs/reference-pulsar-admin.md [hunk 3]
```diff
@@ -1815,6 +1822,36 @@ Subcommands
...
+* `set-max-unacked-messages-per-consumer`
```
Comment (freeznet) [inline site2/docs/reference-pulsar-admin.md:1847]: `set-max-unacked-messages-on-consumer`

File: site2/docs/reference-pulsar-admin.md [hunk 4]
```diff
@@ -1815,6 +1822,36 @@ Subcommands
...
+* `remove-max-unacked-messages-per-consumer`
```
Comment (freeznet) [inline site2/docs/reference-pulsar-admin.md:1848]: `remove-max-unacked-messages-on-consumer`

**Review by freeznet — CHANGES_REQUESTED** [review]
Please refer to my comments, some commands name are not aligned with code.

**Comment by Jennifer88huang** [comment]
@freeznet Thank you for your comments. Is it better to keep consistent for them in the [code](https://github.com/apache/pulsar/blob/master/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java#L1040-L1048)?
image

**Comment by freeznet** [comment]
@Jennifer88huang I think all commands should in lower case and unified format, as for the blue blocks, like `get-maxProducers`, should changed to `get-max-producers`. For the red blocks, I think they are fine, since the command name is just same as the function name, like `GetMaxConsumersPerSubscription` -> `get-max-consumers-per-subscription`.

**Comment by codelipenghui** [comment]
@freeznet I think `get-maxProducers` is mistaken in the previous PR. We can add `get-max-producers` command first and add `@Deprecated` for the non-lower-case commands. And we can also make the non-lower-case command does not appear while run `bin/pulsar-admin topics`. This will guarantee compatibility.

**Comment by Jennifer88huang** [comment]
@freeznet @codelipenghui thank you for your confirmation.
For those in the red boxes, I want to say that in namespaces, we use the word "xxx-per-xxx" (see screenshot below). Since those topic-level polices come from the namespace level, is it more user-friendly if we keep consistent and use the word "per" for topic-level policies? Otherwise, users might use the wrong commands if they want to adopt namespace-level policies to topic-level polices. 
image

**Comment by codelipenghui** [comment]
@Jennifer88huang No, topic policy always for one topic, so we don't need to add -per-topic

**Comment by Jennifer88huang** [comment]
@sijia-w confirmed with eng, we need to keep topic-level commands consistent with that in the namespace-level.
So it should be "get-max-producers", "get-max-unacked-messages-per-consumer", get-max-unacked-messages-per-subscription". Use this kind of format in the docs, and eng will refine the code accordingly.
cc @codelipenghui @freeznet

**Comment by Jennifer88huang** [comment]
Thank you @315157973 for fixing the command name issues in #9215. Shall we go on with the doc PR? @freeznet

**Review by freeznet — APPROVED** [review]
LGTM

**Comment by Jennifer88huang** [comment]
Thank you @freeznet 
@315157973 @codelipenghui @Huanli-Meng do you have any concern on it?

**Review by Huanli-Meng — APPROVED** [review]

**Pull request merged**